python/create_account_with_captcha.py

```diff
@@ -75,13 +90,42 @@ def get_captcha_fields():
             return k and k['fields']
     return None
 
+
+def get_registration_fields():
+    """ Fetch the registration form fields from `authmanagerinfo` module """
+
+    response = S.get(
+        url=MEDIA_API_ENDPOINT,
+        params={
+            'action': 'query',
+            'meta': 'authmanagerinfo',
+            'amirequestsfor': 'create',
+            'format': 'json'})
+
+    data = response.json()
+    query = data and data['query']
+    authmanagerinfo = query and query['authmanagerinfo']
+    fields = authmanagerinfo and authmanagerinfo['requests']
+
+    ret_field = dict()
```

Comment:
In place of `dict()` just use `{}`

Reply:
Also, `ret_field` name could be more readable. May be use fields instead. And, then change the name of fields variable to `requests`.